### DISCUSSION TITLE
v0.18-dev

### DISCUSSION THREAD

**Discussion by sriramveeraghanta** [makeplane/plane#4240] [Announcements]
We are back on pace, introducing v0.18-dev, which will make you fall in love with Plane more and more. Improvements to Pages, a refreshed Kanban layout, scripts for instance admins on self-hosted instances, and much more. Plus, enhanced performance!

### **A whole new experience on Plane Pages**
![pages](https://github.com/makeplane/plane/assets/9498163/639e7d93-4f9a-4181-a5ba-043100f56c05)

We've enhanced Pages to be more beautiful, smooth, and intuitive in a single update. #4090, #4198.

- We've improved the typography—line-heights, margins, paddings, font weights, and sizes; headings; paragraphs; and more—for better readability and consistency.
- Tables as a feature—we've improved our tables with a new UI, row and column configuration, added scrollbars, and more.
- Face-lift to code-blocks—improved paddings, disabled slash commands, decreased tab indentation from 4 to 2, and added an option to copy the contents of code-blocks with a click.
- Support for dark mode—we've improved the colors in dark mode, including code colors, inline code colors, and font colors.
- Auto-merging of lists—ordered and unordered list items, including nested list items, now support auto-merging at every transaction. Tab indentation on lists functions as intended, even after merging two lists.
- Block-based editing—You can now drag and drop content blocks; just hover over a block and rearrange them based on your preferences.

### New Kanban experience
https://github.com/makeplane/plane/assets/9498163/6e62c9d9-33c8-481d-9fce-bbbadaa76744

We've refactored our whole Kanban experience. #4189

- Enjoy quicker and more responsive interactions when dragging and dropping cards.
- We've improved both horizontal and vertical scrolling within the Kanban boards. Now, moving cards across different sections is more fluid.
- Each column now supports independent scrolling. This update allows you to navigate large boards with ease.

### Create Instance Admin with a script

You can now add a script that you can use to create a new instance admin via the command line. Go to your backend shell and use the command: `python manage.py create_instance_admin <email>`. #4210

## **Improvements**

- Lists and kanban cards are completely clickable, to navigate inside the issue detail. #4175, #4093
- We've improved the empty states for various pages across Plane to enhance clarity and education. #4145, #4202
- Bring existing issues onto the Calendar layout from the 'Add' icon on the calendar block. #4094
- Inbox now supports issue attachments. #4190

## **Fixes**

- When Cycles and Modules are disabled from features, they are accessible via routes. This issue is not fixed. #4112, #4121.
- Fixed a data issue with the cycles dropdown on the Issue Create Modal. #4076
- Archived Cycles and Modules, now show analytics. #4191
- Fixed an issue where the API expiry date was always being set to null. It is now set to the assigned time. #4165.
- You can now comment with just people mentions. #4163
- Removed archived cycles and modules from the workspace-level dropdown to streamline the list and eliminate unnecessary items. #4126
- Resolved an issue where navigating from a parent issue to a child issue and back sometimes retained the description of the previous issue. #4157, #4159, #4143
- Fixed UI issues when there are lengthy changes in the activity feed. #4106
- Resolved visibility issue with the selected custom date from the filter dropdown on the project board. #4069
- Removing the Parent issue from the issue edit modal will now detach the parent from the issue. #4103, #4059We are back on pace, introducing v0.18-dev, which will make you fall in love with Plane more and more. Improvements to Pages, a refreshed Kanban layout, scripts for instance admins on self-hosted instances, and much more. Plus, enhanced performance!

Thanks to all our contributors -  @pablohashescobar, @NarayanBavisetti, @aaryan610, @gurusainath, @rahulramesha, @AnmolBhatia1001, @prateekshourya29, @CrsiX, @Palanikannan1437, @paul1278, @ZerNico, @CoolCu, @17347013442.

<hr /><em>This discussion was created from the release <a href='https://github.com/makeplane/plane/releases/tag/v0.18-dev'>v0.18-dev</a>.</em>

**Comment by GxSupport** [comment]
Hello dear programmers. When do you want to add git sync to the local version?

**Comment by inkloop** [comment]
Hello, Plane Team! @sriramveeraghanta

Do we have custom fields on roadmap?

Thank you for advance!

**Reply by gchanchal** [reply]
Hi @danielcolgan , yes we do have Custom Fields, Issue types, Dashboard and many more advanced features in our next offering which is Plane Pro. Details would be out in couple of weeks. Stay tuned!!
